TexasInstruments/SNx4HC595/GHIElectronics.TinyCLR.Drivers.TexasInstruments.SNx4HC595/GHIElectronics.TinyCLR.Drivers.TexasInstruments.SNx4HC595.nuspec

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>GHIElectronics.TinyCLR.Drivers.TexasInstruments.SNx4HC595</id>
+    <version>1.0.0-preview2</version>
+    <authors>GHI Electronics, LLC</authors>
+    <owners>GHI Electronics, LLC</owners>
+    <projectUrl>https://ghielectronics.com/</projectUrl>
+    <iconUrl>http://files.ghielectronics.com/downloads/Logos/GHIElectronicsTinyCLRNuGet.png</iconUrl>
+    <licenseUrl>http://files.ghielectronics.com/downloads/TinyCLR/License.txt</licenseUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>Texas Instruments 8-Bit Shift Registers With 3-State Output Registers driver for TinyCLR OS.</description>
+    <copyright>Copyright GHI Electronics, LLC 2018</copyright>
+    <tags>TinyCLR TinyCLROS TexasInstruments Texas Instruments ShiftRegister Shift Register SNx4HC595 SN54HC595 SN74HC595</tags>
```

Comment:
Just keep the tags `TinyCLR` `TinyCLROS` and `SNx4HC595`

Reply:
Done! I kept the other two because the driver will work the same for all of these: SNx4HC595 SN54HC595 SN74HC595 and it is in line with the STM WiFi driver: https://github.com/ghi-electronics/TinyCLR-Drivers/blob/dev/STMicroelectronics/SPWF04Sx/GHIElectronics.TinyCLR.Drivers.STMicroelectronics.SPWF04Sx.nuspec <tags>TinyCLR TinyCLROS SPWF04SC SPWF04SA SPWF04</tags>

Reply:
Done!